Update meta descriptions for 25 most popular Learn pages (#1370)
astronomer/docs: learn/task-groups.md

---
title: "Airflow task groups"
sidebar_label: "Task groups"
id: task-groups
---

<head>

>   <meta name="description" content="Follow Astronomer’s step-by-step guide to to use task groups for organizing tasks within the graph view of the Airflow user interface." />

Reviewer: ```suggestion
  <meta name="description" content="Learn how you can use task groups to organize tasks within the graph view of the Airflow user interface." />
```
If you don't use this recommendation, update "to to".

>   <meta name="description" content="Follow Astronomer’s step-by-step guide to to use task groups for organizing tasks within the graph view of the Airflow user interface." />

Author: ```suggestion
  <meta name="description" content="Follow Astronomer’s step-by-step guide to use task groups for organizing tasks within the graph view of the Airflow user interface." />
```

>   <meta name="og:description" content="Follow Astronomer’s step-by-step guide to to use task groups for organizing tasks within the graph view of the Airflow user interface." />

Reviewer: ```suggestion
  <meta name="description" content="Learn how you can use task groups to organize tasks within the graph view of the Airflow user interface." />
```
If you don't use this recommendation, update "to to".

>   <meta name="og:description" content="Follow Astronomer’s step-by-step guide to to use task groups for organizing tasks within the graph view of the Airflow user interface." />

Author: ```suggestion
  <meta name="og:description" content="Follow Astronomer’s step-by-step guide to use task groups for organizing tasks within the graph view of the Airflow user interface." />
```
